test/e2e/windows/memory_limits.go

```diff
@@ -217,18 +202,33 @@ func getNodeMemory(ctx context.Context, f *framework.Framework) nodeMemory {
 	}
 
 	nodeMem := nodeMemory{
-		capacity:      status.Capacity[v1.ResourceMemory],
-		allocatable:   status.Allocatable[v1.ResourceMemory],
+		capacity:      node.Status.Capacity[v1.ResourceMemory],
+		allocatable:   node.Status.Allocatable[v1.ResourceMemory],
 		systemReserve: systemReserve,
 		hardEviction:  hardEviction,
-		// these are not implemented and are here for future use - will always be 0 at the moment
-		kubeReserve:  kubeReserve,
-		softEviction: softEviction,
+		kubeReserve:   kubeReserve,
+		softEviction:  softEviction,
 	}
 
 	return nodeMem
 }
 
+// getNodeMemory populates a nodeMemory struct with information from the first
```

Comment:
I don't follow what you mean by "first".

Reply:
Ah I see. You just mean the first element of the list of windows nodes.

Reply:
Let me update that :)